### PR TITLE
adds hyperloglog benchmarks

### PR DESCRIPTION
**Pull request by seut**
also adds a count distinct benchmarks and moves the count benchmarks into the `select/` folder

### CODE REVIEW

File: py/specs/count.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [setup]
```
Comment (chaudum) [inline py/specs/count.toml]: I guess it the intention was to move this file into the select/ folder, then this file should be removed

File: py/specs/select/count.toml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[setup]
+statement_files = ["sql/uservisits.sql"]
```
Comment (chaudum) [inline py/specs/select/count.toml]: if the filepaths are relative to the toml file, then it should start with `../sql`

File: py/tracks/latest.toml [hunk 3]
```diff
@@ -48,5 +48,6 @@ full = [
     "../specs/in_subquery.toml",
     "../specs/in_subquery.toml",
     "../specs/union.toml",
-    "../specs/select/disabled_doc_values.toml"
+    "../specs/select/disabled_doc_values.toml",
+    "../specs/select/hyperloglog.toml"
```
Comment (chaudum) [inline py/tracks/latest.toml]: minor: add a trailing comma, [toml supports that](https://github.com/toml-lang/toml/blob/master/versions/en/toml-v0.4.0.md#array). it has the nice side effect that when you add a line, the previous line does not need to be changed :)

File: py/tracks/latest.toml [hunk 4]
```diff
@@ -27,7 +27,7 @@ full = [
     "../specs/delete_single.py",
     "../specs/delete_bulk.py",
     "../specs/any.py",
-    "../specs/count.toml",
+    "../specs/select/count.toml",
```
Comment (chaudum) [inline py/tracks/latest.toml]: just personal preference: move it down to the other specs that are in the same folder, so they are "grouped"

**Comment by seut** [comment]
@chaudum please check again

**Pull request merged**